stk/calculators/optimization/macromodel.py

```diff
@@ -608,6 +615,46 @@ def __init__(self,
                          timeout=timeout,
                          use_cache=use_cache)
 
+    @staticmethod
+    def _check_params(
+        minimum_gradient,
+        maximum_iterations
+    ):
+        """
+        Check if the optimization parameters are valid for MacroModel.
+
+        MacroModel ``.com`` files have well-defined column widths and
+        special ways to implement too long input. This method raises
+        `MacroModelInputError` exception if the input cannot be
+        converted into a valid ``.com`` file entry.
+
+        Parameters
+        ----------
+        minimum_gradient : :class:`float`, optional
+            The gradient at which optimization is stopped.
+
+        maximum_iterations : :class:`int`, optional
+            The maximum number of iterations done during the
+            optimization.
+
+        Returns
+        -------
+        None : :class:`NoneType`
+
+        """
+
+        # Minimum convergence gradient is `00000.0001`
+        if minimum_gradient < 0.0001:
+            raise MacroModelInputError(
+                'Convergence gradient (< 0.0001) is too small.'
+            )
+
+        # Maximum number of iterations is `999999`
```

Comment:
full stop

Reply:
Removed the comments here (now in docstrings).